app/controllers/imports_controller.rb

```diff
@@ -1,71 +1,51 @@
 class ImportsController < ApplicationController
   before_action :set_podcast
-  before_action :set_import, only: %i[show edit update destroy]
+  before_action :set_import, only: :show
 
   # GET /imports
   def index
-    @imports = [] # Import.all
+    @imports = @podcast.podcast_imports
+    @import = @podcast.podcast_imports.new
+    authorize @import
   end
 
-  # GET /imports/1
   def show
-  end
-
-  # GET /imports/new
-  def new
-    # @import = Import.new
-  end
-
-  # GET /imports/1/edit
-  def edit
+    authorize @import
   end
 
   # POST /imports
   def create
-    # @import = Import.new(import_params)
-
-    # respond_to do |format|
-    #   if @import.save
-    #     format.html { redirect_to import_url(@import), notice: "Import was successfully created." }
-    #   else
-    #     format.html { render :new, status: :unprocessable_entity }
-    #   end
-    # end
-  end
-
-  # PATCH/PUT /imports/1
-  def update
-    # respond_to do |format|
-    #   if @import.update(import_params)
-    #     format.html { redirect_to import_url(@import), notice: "Import was successfully updated." }
-    #   else
-    #     format.html { render :edit, status: :unprocessable_entity }
-    #   end
-    # end
-  end
-
-  # DELETE /imports/1
-  def destroy
-    # @import.destroy
-
-    # respond_to do |format|
-    #   format.html { redirect_to imports_url, notice: "Import was successfully destroyed." }
-    # end
+    @import = @podcast.podcast_imports.new(import_params)
+    authorize @import
+    @import.get_feed
```

Comment:
added this line as per the last comment re: validating the RSS url, which i realized already [happens here](https://github.com/PRX/feeder.prx.org/blob/05d5462625e84778fc7a610a36acfb80b1b0e82a/app/models/podcast_import.rb#L179), so i'm not sure if this call is actually just redundant.